### PR TITLE
added code of conduct explanation

### PR DESCRIPTION
**Pull request by lucacioria**
I've added explanations to every CoC line. The CoC proved to be even more "valuable" than I initially thought.. and I think it deserves some love and attention.

### CODE REVIEW

File: CoC.md [hunk 1]
```diff
@@ -4,11 +4,69 @@ We have a very succinct code of conduct.
...
+All such tools are good, as long as the commitment is written down and visible to both requester and performer.
+
+## Share notes after meetings
+
+A key aspect of good collaboration is alignment among team members. Alignment of goals, alignment of information. Sharing notes after meetings serves two purposes:
+
+1. Inform people who were not present at the meeting
+2. Recap in written format the key outcomes and decisions of the meeting
+
+For project related meetings, use the project mailing lists (typically `project-name@buildo.io`). You can find a list and manage your subscriptions [here](https://groups.google.com/a/buildo.io/forum/#!overview). To update on R&D, use the `buildo/core` issue. You can also use slack for quick updates.
```
Comment (gabro) [inline CoC.md]: this is probably a better interface for managing subscriptions https://gaps.our.buildo.io

**Comment by gabro** [comment]
apart from minor comment, big +1

**Comment by lucacioria** [comment]
@gabro improved

**Comment by gabro** [comment]
🌵 almost moved by merging a PR from you

**Pull request merged**